lib/alchemy_cms.rb

```diff
@@ -15,7 +15,6 @@
 require "jquery-rails"
 require "jquery-ui-rails"
 require "kaminari"
-require "non-stupid-digest-assets"
```

Comment:
I don't understand how it could take the script now that it is called "non_stupid_digest_assets.rb".
Shouldn't I change it to:

`require "non_stupid_digest_assets"` ?

Reply:
That or rename the file.

Reply:
Yes, I've renamed the file from the gem, because I've got a warning from rubocop. I will change that line.